app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala

```diff
@@ -128,20 +131,16 @@ class SchemeMembersListController @Inject()(
                       previousUserAnswers,
                       pages.nonsipp.memberdetails.Paths.personalDetails
                     ) == Updated
-                    logger.info(s"""[ViewOnlyMode] Status for member details list is ${if (updated) "updated"
```

Comment:
It looks like the logging has been removed from this controller, but left in for the other controllers, may I ask why it's been removed here?

Reply:
Yes, I added those in to debug- I forgot to remove the others